firmware/robot2015/src-fpga/CMakeLists.txt

```diff
@@ -0,0 +1,161 @@
+###############################################################################
+################################ LOCAL VARIABLES ##############################
+###############################################################################
+# Device specifics
+set(DEV_PARTNUM xc3s100etq144-4)
+
+# names are limited to 8.3 filename types since we need on-board access to this during runtime
+# lookup the limitations of an 8.3 filename before changing the output bitfile's name if that's ever needed
+set(OUTPUT_FILENAME_BASE rj-fpga)
+
+ # These are set with the above
+set(BIT_OUT_FILENAME ${OUTPUT_FILENAME_BASE}.nib)
+
+# This is the root path to anything Xilinx related
+set(XILINX_SYSTEM_ROOT /opt/Xilinx)
+
+
+###############################################################################
+########################## FIND PATH TO ISE WEBPACK ###########################
+###############################################################################
+# look for installed Xilinx tools in /opt/Xilinx
+message(STATUS "Searching for Xilinx tools in '${XILINX_SYSTEM_ROOT}'")
+file(GLOB XILINX_VERSION_DIRS
+    ${XILINX_SYSTEM_ROOT}/*
+)
+
+# Pick out the folders that look like a floating point formatted value
+string(REGEX MATCHALL
+    "${XILINX_SYSTEM_ROOT}/(([0-9]+)?[\\.]).*"
+    XILINX_VERSION_DIRS
+    "${XILINX_VERSION_DIRS}"
+)
+
+# Remove anything that isn't a directory
+foreach(TMPVAR ${XILINX_VERSION_DIRS})
+    if(NOT((IS_DIRECTORY ${TMPVAR}) AND (EXISTS ${TMPVAR})))
+        list(REMOVE_ITEM XILINX_VERSION_DIRS ${TMPVAR})
+    endif()
+endforeach()
+
+# Find the highester version 
+set(XILINX_ROOT "0.0")
+foreach(TMPVAR ${XILINX_VERSION_DIRS})
```

Comment:
For some reason this is choosing the lowest version available, not the highest...  Haven't looked into why yet.  I'm pretty sure this working before, did you change anything related to this part?